index.html

```diff
@@ -1,38 +1,320 @@
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div>
+              <ul class="card-tags">
+                <li>Ruby on Rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-body">
+            <img
+              src="./images/photo-1533035353720-f1c6a75cd8ab.jpg"
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div>
+              <ul class="card-tags">
+                <li>Ruby on Rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-body">
+            <img
+              src="./images/photo-1533035353720-f1c6a75cd8ab.jpg"
+              alt=""
+              class="card-image"
+            />
+            <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+            <div>
+              <ul class="card-tags">
+                <li>Ruby on Rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+      </section>
+    </main>
+      <section class="about-us">
+        <h1 class="about-heading">About Me</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, do not hestiate to contact me.
+        </p>
+      </section>
+      <button class="card-button">Get My Resume</button>
+      <div class="footer-grid">
+        <section>
+          <div class="footer-card">
+            <div>
+              <img
+                class="footer-img"
+                src="./images/icon-languages.svg"
+                alt=""
+              />
+            </div>
+            <div>
+              <h2>Languages</h2>
+              <div class="group-tag">
+                <div class="footer-list">
+                  <h4 class="footer-tags">JavaScript</h4>
+                  <h4 class="footer-tags">Ruby</h4>
+                </div>
+                <div class="footer-list">
+                  <h4 class="footer-tags">Html</h4>
+                  <h4 class="footer-tags">CSS</h4>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
```

Comment:
- [ ] Section doesn't have any semantic meaning here. Kindly replace that. 
You can check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) when to use a `section` tag
`HINT: You can use a generic div if you want here.`

Reply:
Corrected. Section tag removed and replaced with div.